napari/utils/settings/_defaults.py

```diff
@@ -0,0 +1,89 @@
+"""Settings management.
+"""
+
+from enum import Enum
+from typing import List, Tuple
+
+from pydantic import BaseSettings, Field
+
+from napari.utils.events.evented_model import EventedModel
+
+
+class QtBindingEnum(str, Enum):
+    """Python Qt binding to use with the application."""
+
+    pyside = 'pyside2'
+    pyqt = 'pyqt5'
```

Comment:
I'd remove Enum from the type name, maybe QtBindingChoice? Same below.

Reply:
Fixing